cmd/hperf/main.go

```diff
@@ -239,6 +244,14 @@ func before(ctx *cli.Context) error {
 func parseConfig(ctx *cli.Context) (*shared.Config, error) {
 	shared.DebugEnabled = debug
 
+	concur := ctx.Int(concurrencyFlag.Name)
+	if concur == 0 {
+		concur = runtime.NumCPU() / 2
+		if concur == 0 {
+			concur = 1
+		}
```

Comment:
```suggestion
		concur = max(1, runtime.NumCPU() / 2)
```
Isn't this Go 1.21 or something?